optaplanner-core/src/main/java/org/optaplanner/core/impl/score/director/ScoreDirectorFactoryFactory.java

```diff
@@ -108,16 +109,20 @@ public ScoreDirectorFactoryFactory(ScoreDirectorFactoryConfig config) {
                 .filter(Objects::nonNull).count() > 1) {
             List<String> scoreDirectorFactoryPropertyList = new ArrayList<>(4);
             if (easyScoreDirectorFactory != null) {
-                scoreDirectorFactoryPropertyList.add("an easyScoreCalculatorClass");
+                scoreDirectorFactoryPropertyList
+                        .add("an easyScoreCalculatorClass (" + config.getEasyScoreCalculatorClass().getName() + ")");
             }
             if (constraintStreamScoreDirectorFactory != null) {
-                scoreDirectorFactoryPropertyList.add("a constraintProviderClass");
+                scoreDirectorFactoryPropertyList
+                        .add("a constraintProviderClass (" + config.getConstraintProviderClass().getName() + ")");
             }
             if (incrementalScoreDirectorFactory != null) {
-                scoreDirectorFactoryPropertyList.add("an incrementalScoreCalculatorClass");
+                scoreDirectorFactoryPropertyList.add(
+                        "an incrementalScoreCalculatorClass (" + config.getIncrementalScoreCalculatorClass().getName() + ")");
             }
             if (droolsScoreDirectorFactory != null) {
-                scoreDirectorFactoryPropertyList.add("a droolsScoreDirectorFactory");
+                scoreDirectorFactoryPropertyList
+                        .add("a droolsScoreDirectorFactory (" + getDrlResourcesAndFilesOverview() + ")");
```

Comment:
It might not be clear for everyone that droolsScoreDirectorFactory applies to scoreDrl and scoreDrlFile in their solverConfig.xml, making it much harder to fix the error.

Reply:
Can we do something like this?

```
                        .add("a scoreDrlList (" + ConfigUtils.limit(solverConfig.getScoreDrlList())
                        + ") or a scoreDrlFileList (" + ConfigUtils.limit(solverConfig.getScoreFileDrlList().stream().map(File::getName).toList()) + "));
```